### ISSUE TITLE
Question About Utility

### ISSUE THREAD

**Issue by FrableTech** [winterbe/github-matrix-screensaver#19]
Is there a way to change the source from the github commits to say a local text file? I want to be able to run this without wifi and to be able to change the characters displayed.

**Comment by chriszrc** [comment]
Yeah, I have same question. Is it as easy as updating the ajax calls in the js file? Or does something need to be recompiled
